testsuites/CBLTester/upgrade_tests/test_cbl_upgrade.py

```diff
@@ -94,54 +97,135 @@ def test_upgrade_cbl(params_from_base_suite_setup):
+        (test_multiple_property_fts, (params_for_query_tests, 'content', 'name', 'beauty AND art', 'landmark', True)),
+        (test_multiple_property_fts, (params_for_query_tests, 'content', 'name', '(beauty AND art) OR cafe', 'landmark',
+                                      True)),
+        (test_multiple_property_fts, (params_for_query_tests, 'content', 'name', '(beautiful OR art) AND photograph',
+                                      'landmark', True)),
+        (test_multiple_property_fts, (params_for_query_tests, 'content', 'name', 'restaurant NOT chips', 'landmark',
+                                      True)),
+        (test_multiple_property_fts, (params_for_query_tests, 'content', 'name', 'centre art', 'landmark', False)),
+        (test_multiple_property_fts, (params_for_query_tests, 'content', 'name', 'town*', 'landmark', True)),
+        (test_multiple_property_fts, (params_for_query_tests, 'content', 'name', '^Beautiful', 'landmark', False)),
+        (test_multiple_property_fts, (params_for_query_tests, 'content', 'name', 'name:cafe art', 'landmark', False)),
+        (test_multiple_property_fts, (params_for_query_tests, 'content', 'name', 'beautiful OR arts', 'landmark',
+                                      False)),
+        (test_multiple_property_fts, (params_for_query_tests, 'content', 'name', 'beautiful AND art', 'landmark',
+                                      False)),
+        (test_multiple_property_fts, (params_for_query_tests, 'content', 'name', '(beauty AND splendour) OR food',
+                                      'landmark', False)),
+        (test_multiple_property_fts, (params_for_query_tests, 'content', 'name', 'restaurant NOT chips', 'landmark',
+                                      False)),
+        (test_fts_with_ranking, (params_for_query_tests, 'content', 'beautiful', 'landmark')),
+        (test_getDoc_withValueTypeDouble, (params_for_query_tests, 'doc_with_double_1')),
+        (test_getDoc_withLocale, (params_for_query_tests, 'doc_with_double_1')),
+        (test_query_arthimetic, (params_for_query_tests,))
+    ]
+
+    log_info("\nRunning Query tests")
+    tests_result = OrderedDict()
+    test_passed = 0
+    test_failed = 0
+    for item in query_test_list:
+        query_test_func = item[0]
+        args = item[1]
+        # Running query test one by one
+        log_info("\n")
+        log_info("*" * 20)
+        log_info("Executing {} with arguments {}".format(query_test_func.__name__, args[1:]))
+        try:
+            query_test_func(*args)
+            log_info("PASSED")
+            tests_result["{}{}".format(query_test_func.__name__, args[1:])] = "PASSED"
+            test_passed += 1
+        except Exception as err:
+            log_info("FAILED:\n {}".format(err))
+            tests_result["{}{}".format(query_test_func.__name__, args[1:])] = "FAILED"
+            test_failed += 1
+
+    log_info("\n\nTests Result: PASSED {}, FAILED {}".format(test_passed, test_failed))
```

Comment:
if any query test failed, will it show it in jenkins which tests failed? If not, please make upgrade test failed, even 1 test failed . You can add assert(false) if number of failed tests is not zero

Reply:
Actually, you'll the see the result of each query test failure. I've added the logic to mark test failed if any query tests failed.